fe/fe-core/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java

```diff
@@ -507,12 +511,15 @@ public List<List<Comparable>> getDbInfo() {
             try {
                 DatabaseTransactionMgr dbMgr = getDatabaseTransactionMgr(dbId);
                 runningNum = dbMgr.getRunningTxnNums() + dbMgr.getRunningRoutineLoadTxnNums();
+                totalRunningNum += runningNum;
             } catch (AnalysisException e) {
                 LOG.warn("get database running transaction num failed", e);
             }
-            info.add(runningNum);
+            info.add(String.valueOf(runningNum));
             infos.add(info);
         }
+        List<String> info = Arrays.asList("Total", String.valueOf(dbIds.size()), String.valueOf(totalRunningNum));
```

Comment:
For the last line, better use same schema for each column:
I suggest：
```
0, "Total", runningnum
```
So that the first column is always integer

Reply:
@morningman  This is because in the show proc "/cluster_health/tablet_health" command, it was generated in my current format to maintain consistency. Do you think it still needs to be modified?

Reply:
@morningman done.